### PR TITLE
DOCS-2644: Address Alyssa's feedback

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/use-cases/image-data.md [hunk 1]
```diff
@@ -42,7 +42,8 @@ In the next how-to guide you can use the images you collect here to [train compu
 {{<imgproc src="/icons/components/camera.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="configure a camera component">}}
 **1. Configure a camera**
 
-Configure a camera component, such as a [webcam](/components/camera/webcam/), on your machine.
+On your [machine's page](#prerequisites), configure any camera component.
+If you are nor sure what to use, start with a [webcam](/components/camera/webcam/).
```
Comment (JessamyT) [inline docs/use-cases/image-data.md:46]:
```suggestion
If you are not sure what to use, start with a [webcam](/components/camera/webcam/).
```

File: docs/use-cases/image-data.md [hunk 2]
```diff
@@ -69,18 +70,37 @@ For example, set it to `0.05` to capture an image every 20 seconds.
 
 Set the MIME type to your desired image format, for example `image/jpeg`.
 
+**Click save to start capturing data.**
```
Comment (JessamyT) [inline docs/use-cases/image-data.md:73]: Why is this bold?

File: docs/use-cases/image-data.md [hunk 3]
```diff
@@ -69,18 +70,37 @@ For example, set it to `0.05` to capture an image every 20 seconds.
...
+
 {{% /tablestep %}}
 {{% tablestep %}}
 {{<imgproc src="/services/ml/collect.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
 **4. View data in the Viam app**
 
+{{< alert title="Wait until data appears" color="note" >}}
+Your data will sync at the specified sync interval, which may mean you have to wait for data to appear.
```
Comment (JessamyT) [inline docs/use-cases/image-data.md:81]:
```suggestion
Your data will sync at the specified sync interval, which may mean you have to wait and then refresh the page for data to appear.
```

File: docs/use-cases/image-data.md [hunk 4]
```diff
@@ -95,12 +115,18 @@ The following steps use the [`filtered_camera`](https://github.com/erh/filtered_
 
 Configure an ML model service on your machine that is compatible with the ML model you want to use, for example [TFLite CPU](/services/ml/deploy/tflite_cpu/).
 
-From the **Model** dropdown, select the preexisting model you want to use, or click **Add new model** to upload your own.
+{{% /tablestep %}}
+{{% tablestep link="/services/vision/"%}}
+{{<imgproc src="/services/ml/train.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
+**1. Add an ML model to your machine**
```
Comment (JessamyT) [inline docs/use-cases/image-data.md:121]: Is this a copy paste duplicate?

File: docs/use-cases/image-data.md [hunk 5]
```diff
@@ -110,26 +136,41 @@ From the **Select model** dropdown, select the name of your ML model service (fo
...
+Configure a `filtered-camera` component on your machine, following the [attribute guide in the README](https://github.com/erh/filtered_camera?tab=readme-ov-file#configure-your-filtered-camera) to specify the names of your camera and vision service, and add classification and object detection filters.
 
 {{% /tablestep %}}
 {{% tablestep %}}
 {{<imgproc src="/services/icons/data-capture.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
-**4. Configure data capture and sync on the filtered camera**
+**5. Configure data capture and sync on the filtered camera**
 
-Configure data capture and sync just as you would for a webcam.
+Configure data capture and sync just as you did before.
```
Comment (JessamyT) [inline docs/use-cases/image-data.md:150]:
```suggestion
Configure data capture and sync just as you did before for the physical camera. 
```
[opt]

File: docs/use-cases/image-data.md [hunk 6]
```diff
@@ -110,26 +136,41 @@ From the **Select model** dropdown, select the name of your ML model service (fo
...
+{{% /tablestep %}}
+{{% tablestep %}}
+
+{{<imgproc src="/services/ml/collect.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
+**6. View filtered data in the Viam app**
+
+Once you save your configuration, your filtered images will sync and you can [view those images in the Viam app](/services/data/view/) from the **DATA** tab.
+
+{{< alert title="Wait until data appears" color="note" >}}
+Your data will sync at the specified sync interval, which may mean you have to wait for data to appear.
```
Comment (JessamyT) [inline docs/use-cases/image-data.md:164]:
```suggestion
Your data will sync at the specified sync interval, which may mean you have to wait and then refresh the page for data to appear.
```

File: docs/use-cases/image-data.md [hunk 7]
```diff
@@ -110,26 +136,41 @@ From the **Select model** dropdown, select the name of your ML model service (fo
...
+
+If no data appears after the sync interval, check the [**Logs**](/cloud/machines/#logs) and ensure that the condition for filtering is met.
+You can add a [`transform` camera](/components/camera/transform/) to see detections or classifications live from the [**CONTROL** tab](/cloud/machines/#control).
```
Comment (JessamyT) [inline docs/use-cases/image-data.md:167]: This seems slightly unrelated to this alert. Consider moving/removing or at least adding a new line to make a separate paragraph.

**Review by JessamyT — CHANGES_REQUESTED** [review]
Main thing is to fix the duplicated first step in part 2 and corresponding step numbers after

File: docs/use-cases/image-data.md (hunk 2 above)
Comment (npentrel) [reply docs/use-cases/image-data.md:73]: It's quite far down in the step, so worried people will miss it/forget

Comment (npentrel) [inline docs/use-cases/image-data.md:73]:
```suggestion
Click save to start capturing data.
```

File: docs/use-cases/image-data.md (hunk 3 above)
Comment (npentrel) [inline docs/use-cases/image-data.md:81]:
```suggestion
Once you have saved your configuration changes, your data will sync at the specified sync interval, which may mean you have to wait and then refresh the page for data to appear.
```

File: docs/use-cases/image-data.md (hunk 4 above)
Comment (npentrel) [reply docs/use-cases/image-data.md:121]:
```suggestion
**2. Add an ML model to your machine**
```

File: docs/use-cases/image-data.md [hunk 8]
```diff
@@ -110,26 +136,41 @@ From the **Select model** dropdown, select the name of your ML model service (fo
...
+**6. View filtered data in the Viam app**
+
+Once you save your configuration, your filtered images will sync and you can [view those images in the Viam app](/services/data/view/) from the **DATA** tab.
+
+{{< alert title="Wait until data appears" color="note" >}}
+Your data will sync at the specified sync interval, which may mean you have to wait and then refresh the page for data to appear.
+
+If no data appears after the sync interval, check the [**Logs**](/cloud/machines/#logs) and ensure that the condition for filtering is met.
+You can add a [`transform` camera](/components/camera/transform/) to see detections or classifications live from the [**CONTROL** tab](/cloud/machines/#control).
+{{< /alert >}}
```
Comment (npentrel) [inline docs/use-cases/image-data.md:168]:
```suggestion
Your data will sync at the specified sync interval, which may mean you have to wait and then refresh the page for data to appear.

If no data appears after the sync interval, check the [**Logs**](/cloud/machines/#logs) and ensure that the condition for filtering is met.
You can add a [`transform` camera](/components/camera/transform/) to see detections or classifications live from the [**CONTROL** tab](/cloud/machines/#control).
```

File: docs/use-cases/image-data.md (hunk 7 above)
Comment (npentrel) [reply docs/use-cases/image-data.md:166]: removed the alert around it

File: docs/use-cases/image-data.md [hunk 9]
```diff
@@ -69,18 +70,37 @@ For example, set it to `0.05` to capture an image every 20 seconds.
...
 {{% /tablestep %}}
 {{% tablestep %}}
 {{<imgproc src="/services/ml/collect.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
 **4. View data in the Viam app**
 
+{{< alert title="Wait until data appears" color="note" >}}
+Once you have saved your configuration changes, your data will sync at the specified sync interval, which may mean you have to wait and then refresh the page for data to appear.
+
+If no data appears after the sync interval, check the [**Logs**](/cloud/machines/#logs).
+{{< /alert >}}
```
Comment (JessamyT) [inline docs/use-cases/image-data.md:84]:
```suggestion
Once you have saved your configuration changes, your data will sync at the specified sync interval, which may mean you have to wait and then refresh the page for data to appear.

If no data appears after the sync interval, check the [**Logs**](/cloud/machines/#logs).
```
The alert looks awkward within the tablestep format

File: docs/use-cases/image-data.md [hunk 10]
```diff
@@ -95,12 +115,18 @@ The following steps use the [`filtered_camera`](https://github.com/erh/filtered_
 
 Configure an ML model service on your machine that is compatible with the ML model you want to use, for example [TFLite CPU](/services/ml/deploy/tflite_cpu/).
 
-From the **Model** dropdown, select the preexisting model you want to use, or click **Add new model** to upload your own.
+{{% /tablestep %}}
+{{% tablestep link="/services/vision/"%}}
+{{<imgproc src="/services/ml/train.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
+**2. Add an ML model to your machine**
```
Comment (JessamyT) [inline docs/use-cases/image-data.md:121]:
```suggestion
```
This is duplicated, and then step numbers need changing.

**Review by JessamyT — APPROVED** [review]

**Pull request merged**